code/HabitTracker/app/src/main/java/com/example/habittracker/activities/eventlist/EventDetailActivity.java

```diff
@@ -5,18 +5,26 @@
 
 import static java.security.AccessController.getContext;
 
+import android.content.Context;
```

Comment:
There's missing javadoc for this entire file.

Reply:
@Jie-F Could you add javadoc later?